### PR TITLE
:books: Add example multiple teams one org

### PR DESCRIPTION
**Pull request by dkravetz**
This PR adds a simple example to provide logins to multiple teams within the same given Github Organization using Github OAuth as an identity provider.

### CODE REVIEW

File: docs/pages/admin-guide.mdx [hunk 1]
```diff
@@ -1153,6 +1153,22 @@ spec:
       kubernetes_groups: ["system:masters"]
 ```
 
+Assigning multiple teams within a single organization is as follows:
+
+```yaml
+   teams_to_logins:
+      - organization: octocats
+        team: team-1
```
Comment (klizhentas) [inline docs/pages/admin-guide.mdx:1161]: Looks good. May be have a different target role set for two different teams to highlight the difference?

Comment (dkravetz) [reply docs/pages/admin-guide.mdx:1161]: Makes sense. I updated the example to assign one team to the admin role, and another to the developers role

**Review by russjones — APPROVED** [review]
@klizhentas Looks good to me, can you do a review?

File: docs/pages/admin-guide.mdx [hunk 2]
```diff
@@ -1111,6 +1111,20 @@ spec:
...
+        logins:
+          - admin
+      - organization: octocats
+        team: team-2
+        logins:
+          - developer
+```
+
 <Admonition type="note">
   For open-source Teleport the `logins` field contains a list of allowed
```
Comment (Lourdesvargas77) [inline docs/pages/admin-guide.mdx:1129]: Facebook

**Review by Lourdesvargas77 — CHANGES_REQUESTED** [review]

**Comment by ptgott** [comment]
@russjones @klizhentas The page was removed after v6.2 in favor of an admin section, and there are several examples of `teams_to_logins` in the docs. Should we modify these per the changes in this PR?

**Comment by zmb3** [comment]
Going to close this since v6 is no longer in support.

**Pull request closed without merging**